django/db/models/functions/base.py

```diff
@@ -203,8 +203,7 @@ def __init__(self, expression, substring, **extra):
         """
         if not hasattr(substring, 'resolve_expression'):
             substring = Value(substring)
```

Comment:
@sir-sigurd `output_field` could also work, I suggested using `_output_field` because I knew it would work from my past experience implementing `Func` subclasses.